packages/blade/src/components/Chip/ChipGroup.tsx

```diff
@@ -68,6 +70,24 @@ const _ChipGroup = (
       });
     }
   }
+  const isWrapperLayoutControlled =
+    typeof children === 'object' && getComponentId(children) === 'Box';
```

Comment:
```
            <BaseBox
              display="flex"
              flexDirection="row"
              flexWrap="wrap"
              rowGap={chipGroupGapTokens[size].bottom}
              columnGap={chipGroupGapTokens[size].right}
              marginBottom={chipGroupGapTokens[size].bottom}
            >
              {React.Children.map(children, (child, index) => {
                return <BaseBox key={index}>{child}</BaseBox>; // maybe we can remove the box if needed
              })}
            </BaseBox>
```

This code i shared, should just work without doing any of these checks/conditionals.

Reply:
Yes, but I think we should add a wrapper and check for styles since consumers can pass props like background color or margin and padding, which should not be supported.

Reply:
We should not do arbitrary checks if it's absolutely not necessary. If they add background etc anyways it will look odd. 

Your above code will fail in many cases like so: 

```ts
const SomeCustomWrapperBox = styled(Box)``

<ChipGroup>
  <SomeCustomWrapperBox>
    ...
  </SomeCustomWrapperBox>
</ChipGroup>
``